tests/po_lib_to_return/__init__.py

```diff
@@ -0,0 +1,174 @@
+import attrs
+from url_matcher import Patterns
+
+from web_poet import Injectable, ItemPage, Returns, field, handle_urls, item_from_fields
+
+
+@attrs.define
+class Product:
+    name: str
+    price: float
+
+
+@attrs.define
+class ProductSimilar:
+    name: str
+    price: float
+
+
+@attrs.define
+class ProductMoreFields(Product):
+    brand: str
+
+
+@attrs.define
+class ProductLessFields:
```

Comment:
:smile: 

```suggestion
class ProductFewerFields:
```

Reply:
Good catch! I was having the more/less words in my mind. 😆